content/pt/docs/concepts/cluster-administration/kubelet-garbage-collection.md

```diff
@@ -0,0 +1,71 @@
+---
+reviewers:
+title: Configurando o Garbage Collection do kubelet
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+
+O Garbage collection(Coleta de lixo) é uma função útil do kubelet que limpa imagens e recipientes não utilizados. O kubelet executará o garbage collection para contêineres a cada minuto e para imagens a cada cinco minutos.
+
+Ferramentas externas de garbage collection não são recomendadas, pois podem potencialmente interromper o comportamento do kubelet removendo os recipientes que existem.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Coleta de imagens
+
+O Kubernetes gerencia o ciclo de vida de todas as imagens através do imageManager, com a cooperação do cadvisor.
+
+A política para o garbage collection de imagens leva dois fatores em consideração:
+`HighThresholdPercent` e `LowThresholdPercent`. Uso do disco acima do limite alto acionará o garbage collection. O garbage collection excluirá as imagens que foram menos usadas recentemente até que o nível fique abaixo do limite.
```

Comment:
"Uso do disco acima do limite alto acionará..." to  "Uso do disco acima do limite acionará"